src/Poseidon/Package.hs

```diff
@@ -299,16 +297,9 @@ readPoseidonPackageCollection opts dirs = do
             readFile' filename = withFile filename ReadMode $ \handle -> do
                 theContent <- hGetContents handle
                 mapM return theContent
-    tryDecodePoseidonPackage :: Bool -> (Integer, FilePath) -> PoseidonLogIO (Either PoseidonException PoseidonPackage)
-    tryDecodePoseidonPackage False (numberPackage, path) = do
-        liftIO $ do
-            hClearLine stderr
-            hSetCursorColumn stderr 0
-            hPutStr stderr $ "> " ++ show numberPackage ++ " "
-            hFlush stderr
-        try . readPoseidonPackage opts $ path
-    tryDecodePoseidonPackage True (numberPackage, path) = do
-        liftIO $ hPutStrLn stderr $ "> " ++ show numberPackage ++ ": " ++ path
+    tryDecodePoseidonPackage :: (Integer, FilePath) -> PoseidonLogIO (Either PoseidonException PoseidonPackage)
+    tryDecodePoseidonPackage (numberPackage, path) = do
+        logDebug $ "Package " ++ show numberPackage ++ ": " ++ path
```

Comment:
what happened here? Remove without replacement?

Reply:
Simplification of the progress indicator. It just got a lot simpler and can not be turned on/off any more.